public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -103,6 +84,28 @@ define(function(require) {
     });
   };
 
+  // When there are file events triggered by the editor (or project is renamed), update the publish/republish status
+  Publisher.prototype.showUnpublishedChangesPrompt = function() {
+    var publisher = this;
+
```

Comment:
To make sure that the callback parameter passed in is truly optional, let's set a default value for it here.
`callback = callback || function() {}`